### PR TITLE
Changes the species pay modifier to a 10% reduction.

### PR DESCRIPTION
**Pull request by necromanceranne**
## About The Pull Request

Changes the nonhuman pay modifier to 0.9. Changes the values to use a define.

## Why It's Good For The Game

racism bad ah

After discussion, this is the compromise. Largely this on Arcane's behalf so at least he gets somewhere close to what he wants. If you want to read the old pitch _I made,_ look under the spoiler. I've marked it out to be annoying so nobody has an aneurysm and still thinks this is what the PR is about.

<details>

~~The species payday tax, while thematic to the current state of human supremacy in the game, isn't positive for mechanical play and the ability for people to participate in the economy of the game, and presents problems for development with economy.~~

~~We want people to be able to positively interact with the economy mechanics and have that be a more meaningful aspect of the game. Actively hampering someones financial ability to do so if they happen to be anything but human does not seem like a good idea if that is the goal. It starts to disproportionately affect nonhumans as economy becomes more integrated, and thus results in nonhumans being less able to interact with our content, which is the opposite of what we would want.~~

~~We have a much more thorough method of determining financial ability in the form of job paychecks, which do a plenty good job of encouraging people into roles that have game content (AKA not assistants), and create limitations for various roles without negatively impacting their ability to interact with the economy. The majority of jobs have the same paycheck, and the economy can be based around that standard value. The species tax actually makes it much harder to create a standardization for the economy due to the 25% pay deficit some players may have in a way we can't really control, and thus much more difficult to create consistent content going forward.~~</details>

## Changelog
:cl:
balance: Reduces the nonhuman pay reduction to 10%.
code: Changes pay modifiers to defines for their values.
/:cl:

### CODE REVIEW

**Comment by Jackal-boop** [comment]
Hell yeah

**Comment by R9b1t** [comment]
This undermines our civil liberties as Human mains to OPPRESS the inferiors known as 'other species'

**Comment by OmegaDarkPotato** [comment]
tg has fallen 2023 this is so fucked remember to like and subscribe and hit the bell notification for more

**Comment by optimumtact** [comment]
This is part of the dystopian megacorp feel that we are trying to show with NT.

I'm not sure letting people "engage" with the absolute mess and mockery of an economy we supposedly have is actually better than the benefit above.

I'll have to talk to mothblocks and ninja about it.

**Comment by Jamarkus127** [comment]
> This is part of the dystopian megacorp feel that we are trying to show with NT.
> 
> I'm not sure letting people "engage" with the absolute mess and mockery of an economy we supposedly have is actually better than the benefit above.

Agreeing with oranges. it just means that the sub-species need to work harder to get their dough. you can make it less than 25 percent though. do 13 percent like the gender gap, but with ayylian species.

**Comment by R9b1t** [comment]
Humans should be paid 130% pay in response to this.

**Comment by softcerv** [comment]
> This is part of the dystopian megacorp feel that we are trying to show with NT.
> 
I can get that, although it's also important to discus if having that modifier contributes to that narrative in a meaningful way. 

With my experience playing on a downstream server with the modifier present, it's not something I've really seen contribute to roleplay nor have I seen it discussed ICly.

**Comment by optimumtact** [comment]
best you will get is a reduction to .90 disparity.

**Review by ArcaneMusic — CHANGES_REQUESTED** [review]
I'd take the 90% offer.

Also if you would make these defines that'd be fantastic, that way we never have to think about them again

**Comment by CliffracerX** [comment]
The species pay modifier is bloatcode that adds nothing to the gameplay experience but literal waiting time and further encourages poor behaviour through speciesism.  You can show that NT is a dystopian megacorporation without making people wait a few minutes more to be able to afford a bag of chips at the vendor- and already do with the default lawsets, poor uniform support for non-plantigrade frames, and the official lore blurbs on every species, to speak nothing of all the ways it's shown to suck for *every* employee regardless of their species.

**Comment by YuliceTheDruid** [comment]
> This is part of the dystopian megacorp feel that we are trying to show with NT.
> 
> I'm not sure letting people "engage" with the absolute mess and mockery of an economy we supposedly have is actually better than the benefit above.
> 
> I'll have to talk to mothblocks and ninja about it.

Wouldn't it be better to talk to EOB about it since he's the actual lore head?
Also, even with a lot of this being in lore, don't you think as tg progresses it may be time to phase a lot of it out anyway, given that the playerbase as a whole seems to not even engage with the whole specism thing, and arguably reacts negatively to people who do engage with it? There's also the argument to be made that having it in can attract more people that we ideally wouldn't want as players, but also it'd work as a litmus by keeping it around.

**Comment by Pickle-Coding** [comment]
> The species pay modifier is bloatcode that adds nothing to the gameplay experience but literal waiting time...

Have you ever considered that you are not the only player in the station? Do a favour for someone else and get paid, or even do a bounty. Not that hard.

**Comment by the-orange-cow** [comment]
"Let the be a head of staff" they said.  "Slippery slope is a fallacy" they said.

**Comment by SomeoneYouProbablyKnow** [comment]
> The species pay modifier is bloatcode that adds nothing to the gameplay experience but literal waiting time and further encourages poor behaviour through speciesism.

By that logic we should make all jobs pay the same, heads and assistants. It only encouraged poor behavior after all

**Comment by Dorsisdwarf** [comment]
its 25% less, crew pay is already VERY high in comparison to station prices, and there's hardly anything to buy for most roles. If you really need extra cash for a gimmick do some bounties or just... ask another crewman for money instead of relying on your norp free bucks. You'd be astonished how happy random crew are to just give you credits, especially if you got a reason.

File: code/modules/mob/living/carbon/human/species_types/monkeys.dm
Comment (Jacquerel) [inline code/modules/mob/living/carbon/human/species_types/monkeys.dm:40]: If this PR is being updated to simply change the value, make sure you also put this back.

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Comment by atlasle** [comment]
The only way this could get approved is if the economy actually had an impact ingame, right now money serves insanely little purpose so work on that first, give it importance and actual worth for players

**Comment by Jacquerel** [comment]
> Wasn't our policy with other resource systems, like power generation, to standardize them and _then_ make them matter?

Yes but also these inputs don't really matter because headcoders have already weighed in on what needs to be done for this to be merged.
"The only way this could be approved" isn't a mystery it is spelled out a few comments above here.

**Comment by necromanceranne** [comment]
> This PR absolutely feels like the badly justified result of losing an argument in a voicechat

Actually, the reason is that I saved arcane having to do the PR himself tomorrow and I also thought it was a bad idea. I literally posted the entire discussion in the body of the PR. Don't overthink this.

File: code/modules/mob/living/carbon/human/species_types/monkeys.dm
Comment (necromanceranne) [reply code/modules/mob/living/carbon/human/species_types/monkeys.dm:40]: Has its own define.

**Comment by ExcessiveUseOfCobblestone** [comment]
Is the issue that we base things off of default cost so you run the risk of forgetting/not properly considering the impact to nonhumans?

you’re better off making nonhumans the baseline and just giving humans a raise, that way anytime we have to look at values we are looking at the most difficult path vs rn where we look at the best path and potentially screw over races in the process.

Im pro human bias, I think the issue could be fixed if we just visualized values differently in the code.

**Comment by necromanceranne** [comment]
> Is the issue that we base things off of default cost so you run the risk of forgetting/not properly considering the impact to nonhumans?

You've hit the nail on the coffin.

> you’re better off making nonhumans the baseline and just giving humans a raise, that way anytime we have to look at values we are looking at the most difficult path vs rn where we look at the best path and potentially screw over races in the process.

That's actually not a bad idea tbh. I'll let Arcane weigh on that, maybe I'll close the PR and let him do something more thorough if he wants to adjust prices for the economy with respect to that.

**Comment by moocowswag** [comment]
What if we made the mechanic that selectively pays everyone based on their species into a machine in the vault that way there is an ic way to impact it.

**Comment by carlarctg** [comment]
> What if we made the mechanic that selectively pays everyone based on their species into a machine in the vault that way there is an ic way to impact it.
> 
> Basically a physical bank server that keeps track of how much money someone has been given and how much has been with held from them.

This is fucking genius. It solves every issue excellently.

You can even keep the 'nonhuman tax' by making it so that in CODE the humans get a plus to their paycheck, but in the GAME it looks as if the nonhumans lose it.

```
#define HUMAN_EXTRA_PAYCHECK *1.1

var/paycheck = 200
var/bonus_money_multiplier = HUMAN_EXTRA_PAYCHECK/2 // Lizards get less money. Fuck them.
var/tax_flavor = list("Cold-Blooded Accomodations" = 0.3, "Tiziran Government = 0.6", "NT Lizard Uplifting" = 0.1)

--- 
Paycheck processed.
+250 credits
-4 credits due to Tiziran Government tax.
-6 credits due to NT Lizard Uplifting.
-2 credits due to Cold-Blooded Accomodations.

```

**Comment by necromanceranne** [comment]
I'm bouncing this PR back to arcane to do his thing with so time to close.

Pack it up folk, nothing to see here.

**Pull request closed without merging**
